### PR TITLE
Add `nvtop`, cv2, python packages tests

### LINKED ISSUE #123: Install `nvtop`
This package may require installation from sources.
See https://github.com/Syllo/nvtop

### PR DESCRIPTION
**Pull request by YevheniiSemendiak**
closes #123 , #107

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -35,31 +22,33 @@ RUN APT_INSTALL="apt-get install -y --no-install-recommends" && \
...
         cmake \
         nano \
         tmux \
         htop \
         ssh \
-        # OpenCV
-        libsm6 libxext6 libxrender-dev \
         && \
+        # NVTop >>
+        git clone --depth 1 --branch 1.2.2 -q https://github.com/Syllo/nvtop.git nvtop && \
```
Comment (anayden) [inline Dockerfile:34]: Why `--depth 1` as opposed to `--depth 0`?

File: Dockerfile [hunk 2]
```diff
@@ -68,12 +57,13 @@ COPY requirements/python.txt /tmp/requirements/python.txt
...
     $PIP_INSTALL pip && \
     $PIP_INSTALL -r /tmp/requirements/python.txt -f https://download.pytorch.org/whl/torch_stable.html && \
-    rm /tmp/requirements/python.txt && \
+    rm -r /tmp/requirements && \
 # ==================================================================
 # VSCode server
 # ------------------------------------------------------------------
-    wget https://github.com/cdr/code-server/releases/download/v3.9.1/code-server_3.9.1_amd64.deb  && \
-    dpkg -i code-server_3.9.1_amd64.deb
+    wget -q https://github.com/cdr/code-server/releases/download/v3.9.1/code-server_3.9.1_amd64.deb  && \
```
Comment (anayden) [inline Dockerfile:64]: Since you're updating the code, let's use `v3.11.1`

File: Makefile [hunk 3]
```diff
@@ -30,29 +30,17 @@ image_pip_list:
 e2e_neuro_push:
 	neuro push $(TARGET_IMAGE_NAME):built-$(BASE_IMAGE_TYPE) $(TEST_IMAGE_NAME):$(BASE_IMAGE_TYPE)
 
-TEST_PRESET=cpu-small
-TEST_CMD=
-.PHONY: _test_e2e
-_test_e2e:
+TEST_PRESET=gpu-small
```
Comment (anayden) [inline Makefile:33]: AFAIK we don't have GPUs in dev, so this will most likely fail

**Review by anayden — APPROVED** [review]
Thanks for updating the dependencies and making the environment much more modern! A few super-minor comments.

File: Makefile (hunk 3 above)
Comment (YevheniiSemendiak) [reply Makefile:33]: we're testing it against staging's `default`, there we have `gpu-small` preset.
as for me, that preset is more likely to appear in the cluster than k80 :)

Comment (anayden) [reply Makefile:33]: ah, I see. missed the default cluster part. thanks!

File: Dockerfile (hunk 1 above)
Comment (YevheniiSemendiak) [reply Dockerfile:34]: good question, I've just tested it:
image
its just the number of commits to fetch

Comment (anayden) [reply Dockerfile:34]: Wow, I was sure `--depth 0` works. Thanks for testing!

File: Dockerfile (hunk 2 above)
Comment (YevheniiSemendiak) [reply Dockerfile]: done in 
5c410df

**Pull request merged**